lib/apiservers/service/swagger.json

```diff
@@ -828,6 +828,9 @@
                 "https": {
                   "type": "string",
                   "format": "uri"
+                },
+                "no_proxy": {
+                  "type": "string"
```

Comment:
It seems like this should take an array of URIs; we use "rich" data types in the API even if they're not feasible for the CLI. (This will require some changes to the API handler logic.)

Reply:
According to the snippet above, no_proxy includes not only URIs, but also domain names.

Reply:
Right, URIs can parse '*' or domain names too. I just updated the type to URI array.